Userland/Libraries/LibJS/Tests/builtins/Temporal/Now/Now.plainDate.js

```diff
@@ -34,6 +34,21 @@ describe("correct behavior", () => {
             }
         }
     });
+
+    test("cannot have a time zone with more than a day", () => {
+        [86400000000000, -86400000000000, 86400000000001, 86400000000002].forEach(offset => {
+            const calendar = new Temporal.Calendar("iso8601");
+            const timeZone = {
+                getOffsetNanosecondsFor() {
+                    return offset;
+                },
+            };
+            expect(() => Temporal.Now.plainDate(calendar, timeZone)).toThrowWithMessage(
+                RangeError,
+                "Invalid offset nanoseconds value, must be in range -86400 * 10^9 to 86400 * 10^9"
```

Comment:
This is a lie now

Reply:
Yeah I thought we could just say the range is exclusive now but it isn't very clear.
I added +1 and -1, but I'm not super happy with it.
If you have a better suggestion let me know!

Reply:
Hold on splitting the error

Reply:
updated message seems fine :)